src/components/activity/ActivityDirectiveChangelog.svelte

```diff
@@ -0,0 +1,365 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import HistoryIcon from '@nasa-jpl/stellar/icons/history.svg?component';
+  import { createEventDispatcher, onMount } from 'svelte';
+  import { plan } from '../../stores/plan';
+  import type {
+    ActivityDirective,
+    ActivityDirectiveRevision,
+    ActivityDirectivesMap,
+    ActivityType,
+  } from '../../types/activity';
+  import type { User } from '../../types/app';
+  import type { ArgumentsMap } from '../../types/parameter';
+  import effects from '../../utilities/effects';
+  import { permissionHandler } from '../../utilities/permissionHandler';
+  import { featurePermissions } from '../../utilities/permissions';
+  import { convertUsToDurationString } from '../../utilities/time';
+  import { tooltip } from '../../utilities/tooltip';
+
+  const dispatch = createEventDispatcher();
+  const updatePermissionError = 'You do not have permission to update this activity';
+
+  export let activityDirective: ActivityDirective;
+  export let activityDirectivesMap: ActivityDirectivesMap = {};
+  export let activityTypes: ActivityType[] = [];
+  export let modelId: number;
+  export let user: User | null;
+
+  let hasUpdatePermission: boolean = false;
+  $: if (user !== null && $plan !== null) {
+    hasUpdatePermission = featurePermissions.activityDirective.canUpdate(user, $plan, activityDirective);
+  }
+
+  let activityRevisions: ActivityDirectiveRevision[];
+  $: activityRevisions = [];
+
+  let activityRevisionChangeMap: { currentValue: string; name: string; previousValue: string }[] = [];
+  $: activityRevisionChangeMap = [];
+
+  $: activityType =
+    (activityTypes ?? []).find(({ name: activityTypeName }) => activityDirective?.type === activityTypeName) ?? null;
+
+  let effectiveRevisionArguments: (ArgumentsMap | undefined)[];
+  $: effectiveRevisionArguments = [];
+
+  function formatDate(dateString: string) {
```

Comment:
Should use same formatting as the sim and plan snapshot time labels which we almost are here but there's a difference in usage of the comma 
<img width="311" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/f3a2a750-5734-4669-863d-9cddb0e68320">
<img width="135" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/1d949fce-82e0-4a2a-a6fd-050f8dcc6f73">
@lklyne